src/lib/edge-functions/registry.mjs

```diff
@@ -321,50 +377,69 @@ export class EdgeFunctionsRegistry {
   }
 
   /**
-   * @param {EdgeFunction} func
+   * Logs an event associated with functions.
+   *
+   * @param {EdgeFunctionEvent} event
+   * @param {object} data
+   * @param {Error} [data.buildError]
+   * @param {string} [data.functionName]
+   * @param {string[]} [data.warnings]
+   * @returns
    */
-  #logAddedFunction(func) {
-    log(`${NETLIFYDEVLOG} ${chalk.green('Loaded')} edge function ${chalk.yellow(this.#getDisplayName(func.name))}`)
-  }
+  #logEvent(event, { buildError, functionName, warnings = [] }) {
```

Comment:
I have centralised all log messages in this method, similar to what we're doing with the serverless functions registry.